src/data/queries/NewsStoryPage.ts

```diff
@@ -0,0 +1,34 @@
+import { QueryData, QueryOpts, QueryParams } from "@next-drupal/query"
```

Comment:
This is based on the example at https://shadcn.notion.site/next-drupal-query-bdbe85efca244b33b7b253ef87938826#:~:text=Bash-,Create,-your%20first%20query. A couple differences:
* we are defining `NewsStoryPageProps` elsewhere, since it's essentially a contract for what `NewsStoryPage` data should look like
* We also defined the mapping above.